src/main/java/Baron.java

```diff
@@ -0,0 +1,92 @@
+import java.util.Scanner;
+
+public class Baron {
+    public static void displayCommandsList(Task[] commandsList) {
+        int taskNumber = 1;
+        System.out.println("\tHere are the tasks in your list:");
+        for (Task command: commandsList) {
+            if (command == null) {
+                break;
+            }
+            System.out.println("\t" + taskNumber + "." + command.getTaskDetails());
+            taskNumber += 1;
+        }
+        System.out.println();
+    }
+
+    public static void main(String[] args) {
+        greetUser();
+        getUserInput();
+        printExitMessage();
+    }
+
+    private static void greetUser() {
+        System.out.println("Hello! I'm Baron");
+        System.out.println("What can I do for you?\n");
+    }
+
+    private static void printExitMessage() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    private static void getUserInput() {
+        Task[] tasks = new Task[100];
+        int numberOfTasks = 0;
+        while (true) {
+            Scanner userInput = new Scanner(System.in);
```

Comment:
a new Scanner object is instantiated every iteration of the while loop, which may cause an unneccessary amount of Scanners to be created